api/user.go

```diff
@@ -122,33 +124,33 @@ func (h *userHandler) Delete() http.HandlerFunc {
 
 }
 
+func (h *userHandler) GetAll() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		users, err := h.us.GetAllUsers()
+		if err != nil {
+			log.Printf("apiUserGETHandler, GetAllUsers(), %s", err.Error())
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		var urs = make([]*UserResp, 0)
+		for _, u := range users {
+			ur := &UserResp{Name: u.Name, Role: u.Role}
+			urs = append(urs, ur)
+		}
+
+		w.WriteHeader(http.StatusOK)
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(urs)
```

Comment:
Can you go through and make sure all of these calls check for err?

https://github.com/ImageWare/TLSential/pull/41/files#diff-85286e6fc71c8ed48b27c838fb8e7437R151

Reply:
Pushed an update. I believe I got them all